src/css/lib/monster.ui.scss

```diff
@@ -458,21 +458,39 @@ div.monster-password-strength {
 	margin-top: 5px;
 	border: solid 1px #c0c0c9;
 	border-radius: 2px;
+
+	.monster-password-strength {
+		&-bar {
+			font-size: 12px;
+			-webkit-box-sizing: border-box;
+			-moz-box-sizing: border-box;
+					box-sizing: border-box;
```

Comment:
No need for vendor specific prefixes, `box-sizing` is [supported ](https://caniuse.com/#search=box-sizing)by all major browsers, all the to IE8.